### ISSUE TITLE
"The given key was not present in the dictionary." error when diffing files

### ISSUE THREAD

**Issue by deadlydog** [deadlydog/VS.DiffAllFiles#15]
In VS 2017 I have noticed that sometimes I will get the following error when trying to diff some files:
```
Unexpected Error Occurred:
The given key was not present in the dictionary.
```
![image](https://user-images.githubusercontent.com/1187140/30308216-0f02ca94-9741-11e7-8c8f-caf5aaa0c64b.png)

I can still compare them the regular way without Diff All Files, and after committing the files, other changes compare without any issue when using Diff All Files.

**Comment by deadlydog** [comment]
I just updated the VS 2017 extension to v1.0.2 which includes the latest version of LibGit2Sharp (v0.26). I had a git repo where I could reliably reproduce the "The given key was not present in the dictionary." error, and it looks to have fixed it. Right now the fix is only in the VS 2017 version of the extension as I'll need to install VS 2015 in order to be able to build the VS 2015 version of the extension (sigh). Anyhow, I do believe this error is resolved :)

**Comment by deadlydog** [comment]
This should actually be fixed in v1.0.3 of the VS 2017 extension.
